data/SampleData/Ccda/170.314B2_Amb_CCD.ccda

```diff
@@ -1027,7 +1027,7 @@
 													<originalText>
 														<reference value="#ID0EBALACA"/>
 													</originalText>
-													<translation xsi:type="CD" code="2670" codeSystem="2.16.840.1.113883.6.96" codeSystemName="SNOMED CT" displayName="Codeine"/>
+													<translation xsi:type="CD" code="261000" codeSystem="2.16.840.1.113883.6.96" codeSystemName="SNOMED CT" displayName="Codeine phosphate"/>
```

Comment:
Would you mind pasting the spec referenced by this update here?

Reply:
Sure, the reference is [here](https://www.hl7.org/fhir/valueset-medication-codes.html)

Reply:
Got it, thanks.